extensions/dash/src/create-docset-link.tsx

```diff
@@ -0,0 +1,32 @@
+import { DashArguments } from "./types";
+import { Action, ActionPanel, Detail, List, showToast, Toast } from "@raycast/api";
+import { createDeeplinkForDocset } from "./utils";
+
+export default function Command(props: { arguments: DashArguments }) {
+  const { docset = "" } = props.arguments;
+  if (!docset) {
+    showToast({
+      style: Toast.Style.Failure,
+      title: "Docset keyword is required",
+    });
+    return <Detail markdown={`Can not create Quicklink, docset keyword is required!`}></Detail>;
+  }
```

Comment:
```suggestion
  const { docset } = props.arguments;
```

Reply:
If it's still a view command, no need to show a simple message? I define it as `required` in package.json, but I can still open the command by press enter while I enter nothing in the command query, is it possible to reject the open command (like in the `no-view` command to just show a toast in the bottom left corner without opening the command) when the query is required?